blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/cloud/model/BbCloudTeam.java

```diff
@@ -30,17 +30,17 @@ public BbCloudTeam(@Nonnull @JsonProperty("username") String teamName,
         }
     }
 
-    public BbCloudTeam(String teamName,
+    public BbCloudTeam(String uuid,
```

Comment:
Does it also need any changes in the UI, it might get used in the pipeline creation wizard. I also wonder if username field should be preserved, their migration guide is clear about preserving username in the URL but silent about whether response will have username removed.

Reply:
I'll have to look more monday, We are calling getUser() which returns who they are, but there's probably at least one place that calls getUser("name")